AnswerGeneratorMicroservice/README.md

```diff
@@ -0,0 +1,13 @@
+Microservice app
```

Comment:
name toooo general, you should name wisely what is the micro-service you are implementing